### DISCUSSION TITLE
More than one manager-type pillar exists

### DISCUSSION THREAD

**Discussion by ccpaasmyser** [Security-Onion-Solutions/securityonion#8584] [Unsupported Versions] [answered]
So i got security onion setup on an install of Cent OS 7, and when I ran the security onion setup it failed the first time, I re-ran the installer and it seems to have installed just fine, however. I am now getting the below error when trying to go in and see and set disabled rules.

[root@securityonion ~]# sudo so-rule disabled list
(This should not happen, the system is in an error state if you see this message.)

More than one manager-type pillar exists, minion id's listed below:
  "/opt/so/saltstack/local/pillar/minions/nettap-0_standalone.sls", "/opt/so/saltstack/local/pillar/minions/securityonion_standalone.sls"
[root@securityonion ~]#

I might add that the interface is very very slow. It seems to take actual minutes for things to load. Any help with the above error would be great.

**Comment by dougburks** [comment, marked as answer]
You should only have one `*_standalone.sls` file in `/opt/so/saltstack/local/pillar/minions/`. It should match the current hostname of the box. You should be able to remove the one that doesn't match the current hostname of the box. If all else fails, you can always perform a full wipe and reinstall.

**Reply by ccpaasmyser** [reply]
fair enough, because its a fresh install to being with i think i'll just do a wipe and fresh load.

**Reply by ccpaasmyser** [reply]
@dougburks so i wiped and reinstalled and the same thing happened as far as the security onion install craps out about 25% through the install. I believe i captured where the erros exist that is causing my issue, but not suer why it errors here.

[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/9425546/sosetup.log)

What i did in the meantime is to rerun the installer and keep all the settings the same, where last time i changed the hostname. Will this cause issues in the system or will it just overwrite things and replace things as necessary? Will this cause issues in the long run?

**Reply by dougburks** [reply]
Re-running the installer **_should_** overwrite and replace as necessary.

**Reply by ccpaasmyser** [reply]
ok, thats what i did.
